stumpwm.lisp

```diff
@@ -219,6 +273,16 @@ The action is to call FUNCTION with arguments ARGS."
        (let ((io (make-instance *default-io-loop*)))
          (io-loop-add io (make-instance 'stumpwm-timer-channel))
          (io-loop-add io (make-instance 'display-channel :display *display*))
+
+         ;; If we have no implementation for the current CL, then
+         ;; don't register the channel.
+         #+sbcl
```

Comment:
The answer is *no*. I forgot about the fact that the non-dummy IO loop is currently SBCL only.